src/main/java/Wessy.java

```diff
@@ -194,22 +233,38 @@ static void printAdded(Task task) {
 
     static void println(String str) {
         int length = str.length();
-        String message = "   |   " + str;
-        for (int i = 0; i < 70 - length - 3; i++) {
-            message += " ";
+        if (length <= 64) {
+            String message = "   |   " + str;
+            for (int i = 0; i < 67 - length; i++) {
+                message += " ";
+            }
+            message += "|";
+            System.out.println(message);
+        } else {
+            System.out.println("   |   " + str.substring(0, 64) + "   |");
+            int remainingLength = length - 64;
+            int leftover = remainingLength % 62;
+            int n = (int) Math.floor(remainingLength/62);
```

Comment:
Might want to split this function into several to make it easier to edit in the future.